<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>1220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="Profiles">
      <property name="title">
       <string>Profiles</string>
      </property>
      <widget class="QComboBox" name="profileComboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>401</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>110</y>
         <width>291</width>
         <height>241</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="newProfileButton">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="copyProfileButton">
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteProfileButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="Configuration">
      <property name="title">
       <string>Configuration</string>
      </property>
      <widget class="QPushButton" name="saveProfileButton">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>30</y>
         <width>171</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Profile</string>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>251</width>
         <height>371</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Venomica</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Client ID:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Venomica</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>State:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Venomica</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Details:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Venomica</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Image Key:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Venomica</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Large Text:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Venomica</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Small Image Key:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>Venomica</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Small Text:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Venomica</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Interval:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>30</y>
         <width>181</width>
         <height>371</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="clientIdEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="stateEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="detailsEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="largeImageEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="largeTextEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="smallImageEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="smallTextEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="intervalSpinBox"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Settings</string>
      </property>
      <widget class="QLabel" name="statusLabel">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>50</y>
         <width>161</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Venomica</family>
         <pointsize>24</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>140</y>
         <width>551</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="startButton">
          <property name="text">
           <string>▶ Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="text">
           <string>■ Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="themeButton">
          <property name="text">
           <string>Dark Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exitButton">
          <property name="text">
           <string>✖ Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPresenceUI">
    <property name="title">
     <string>PresenceUI</string>
    </property>
   </widget>
   <addaction name="menuPresenceUI"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
